src/ui/server/middleware/integrity/index.ts

```diff
@@ -0,0 +1,30 @@
+import { Request, Response } from '../../../types';
+import { INTEGRITY } from '../../content-strings';
+
+const { GOVUK, FORM, COOKIES, GA, MOJ, ACCESSIBILITY } = INTEGRITY;
+
+/**
+ * Middleware function that adds integrity values to the res.locals.SRI object.
+ * These integrity values are used for sub resource Integrity (SRI) checks in web applications.
+ *
+ * @param req - The request object containing information about the incoming HTTP request.
+ * @param res - The response object used to send the HTTP response.
+ * @param next - A callback function to pass control to the next middleware function.
+ */
+export const integrity = (req: Request, res: Response, next: () => void) => {
+  const SRI = {
+    ACCESSIBILITY,
+    MOJ,
+    GOVUK,
+    FORM,
+    COOKIES,
+    GA,
+  };
+
+  res.locals.SRI = {
+    ...res.locals.SRI,
+    ...SRI,
+  };
```

Comment:
I think it can just be `res.locals.SRI = SRI;`, not a blocker

Reply:
Have pushed.